bylaws/officers.md

```diff
@@ -1,18 +1,54 @@
 # Officers of the Corporation
 ## Offices held
-The officers of this corporation shall be a Secretary Functionary, a Treasurer Functionary, and at least one Facilitation Functionary.
+The officers of this corporation shall be a number of Functionaries, with a minimum of a Secretary Functionary and Treasurer Functionary at all times. The corporation, at the board's discretion, may also have a chairman of the board.
```

Comment:
CA law says:

> 5213.  (a) A corporation shall have (1) a chair of the board, who
may be given the title chair, chairperson, chairman, chairwoman,
chair of the board, chairperson of the board, chairman of the board,
or chairwoman of the board, or a president or both,

So in light of that we don't need a president. I'm in favor of having as few people with titles of supreme unchecked power as we can.